src/platform_impl/windows/window_state.rs

```diff
@@ -32,6 +32,7 @@ pub struct WindowState {
     pub modifiers_state: ModifiersState,
     pub fullscreen: Option<Fullscreen>,
     pub is_dark_mode: bool,
```

Comment:
I feel like for the more code simplicity we can use `Theme` instead of `bool` here, so `try_dark_mode` will also return a `Theme`, and then you'd have less mapping to `bool -> Theme` , issuing a new `ThemeChanged` when `current_theme != new_theme`?

Reply:
Yeah I like this changed. Implemented it!